### PR TITLE
Feature ary list

### PR DESCRIPTION
**Pull request by thenav56**

### CODE REVIEW

File: frontend/src/redux/reducers/domainData/assessmentRegistry.js [hunk 1]
```diff
@@ -11,12 +12,19 @@ export const setAryTemplateAction = ({ template }) => ({
...
+    aryFilterOptions,
+});
+
 // HELPERS
 
 // REDUCER
 
 const setAryTemplate = (state, action) => {
     const { template } = action;
+    console.warn(template);
```
Comment (tnagorra) [inline frontend/src/redux/reducers/domainData/assessmentRegistry.js]: Stray console.warn found

File: frontend/src/redux/reducers/domainData/assessmentRegistry.js [hunk 2]
```diff
@@ -27,10 +35,23 @@ const setAryTemplate = (state, action) => {
     return update(state, settings);
 };
 
+const setAryFilterOptions = (state, action) => {
+    const { projectId, aryFilterOptions } = action;
+    const settings = {
+        aryFilterOptions: {
+            [projectId]: { $autoArray: {
```
Comment (tnagorra) [inline frontend/src/redux/reducers/domainData/assessmentRegistry.js]: Is this supposed to be an array?

File: frontend/src/views/Arys/FilterArysForm.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+        const { createdBy } = aryFilterOptions;
+
+        return (
+            <Form
+                ref={(elem) => { this.formComponent = elem; }}
+                className={`arys-filters ${className}`}
+                successCallback={this.handleSubmit}
+                changeCallback={this.handleChange}
+                elements={this.formElements}
+                value={formValues}
```
Comment (tnagorra) [inline frontend/src/views/Arys/FilterArysForm.js]: These need to use NEW Form now :D

File: frontend/src/redux/reducers/domainData/assessmentRegistry.js (hunk 1 above)
Comment (thenav56) [reply frontend/src/redux/reducers/domainData/assessmentRegistry.js]: Removed

File: frontend/src/redux/reducers/domainData/assessmentRegistry.js (hunk 2 above)
Comment (thenav56) [reply frontend/src/redux/reducers/domainData/assessmentRegistry.js]: Fixed

File: frontend/src/views/Arys/FilterArysForm.js (hunk 3 above)
Comment (thenav56) [reply frontend/src/views/Arys/FilterArysForm.js]: Fixed

**Pull request merged**